src/test/test-umount.c

```diff
@@ -45,7 +46,10 @@ static void test_swap_list(const char *fname) {
         }
 
         LIST_HEAD_INIT(mp_list_head);
-        assert_se(swap_list_get(fname, &mp_list_head) >= 0);
+        r = swap_list_get(fname, &mp_list_head);
+        if (r == -EACCES)
```

Comment:
maybe actually use ERRNO_IS_PRIVILEGE() for these kinds of tests, for compat with environments where this causes EPERM…